docs/getting-started/installation.md

```diff
@@ -25,6 +25,29 @@ docker run -d \
 
 {% endtab %}
 
+{% tab title="Compose" %}
+
+docker-compose.yml
```

Comment:
```suggestion
**docker-compose.yml:**

```

We bold headings in tabs elsewhere, so seems appropriate to do the same for the filename here.